### PR TITLE
New Prop Added - expandableCloseAllOnExpand

### PR DESCRIPTION
**Pull request by arzebmscit15**
New props added with named expandableCloseAllOnExpand and type boolean to close other expanded row on expand another row.

### CODE REVIEW

File: src/DataTable/DataTable.tsx [hunk 1]
```diff
@@ -162,12 +171,25 @@ function DataTable<T>(props: TableProps<T>): JSX.Element {
 	const currentTheme = React.useMemo(() => createStyles(customStyles, theme), [customStyles, theme]);
 	const wrapperProps = React.useMemo(() => ({ ...(direction !== 'auto' && { dir: direction }) }), [direction]);
 
+	// const flages = React.useMemo(() => ({ ...(direction !== 'auto' && { dir: direction }) }), [direction]);
```
Comment (bikkimahato) [inline src/DataTable/DataTable.tsx:174]: Commented code should not be pushed to GitHub.

**Comment by ShaneHudson** [comment]
I really appreciate that this feature is being added! @bikkimahato when would you expect the next release to be?

File: src/DataTable/DataTable.tsx (hunk 1 above)
Comment (arzebmscit15) [reply src/DataTable/DataTable.tsx]: Commented codes are removed. Thank You.

**Comment by ShaneHudson** [comment]
Could this PR please be merged in and released on npm?

**Comment by ShaneHudson** [comment]
@jbetancur Hi John just wondered what's the ETA on this PR?

**Comment by ShaneHudson** [comment]
Please do not close this issue.

**Pull request closed without merging**